docs/self-managed/identity/user-guide/tenants/managing-tenants.md

```diff
@@ -0,0 +1,84 @@
+---
+id: managing-tenants
+title: "Managing tenants"
+sidebar_label: "Managing tenants"
+description: "Learn about the methods to manage tenants within the Identity application."
```

Comment:
This is good -- but I would probably say something about _why_ I should manage tenants, for example: "Manage tenants within Identity to XYZ" -- also, we want to refer to Identity as a component 👍